Cargo.lock

```diff
@@ -89,9 +89,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.57"
+version = "1.0.58"
```

Comment:
Is this newer version of `anyhow` necessary? If so, I think it should probably be reflected in the minimum version requirement in the appropriate `Cargo.toml`. Otherwise can you revert this change to `Cargo.lock`?

If you do need the upgrade, I don't think we should be adding cargo-vet exceptions. I hope a delta certification isn't a big deal for that crate.

If you're just adding the exception as a temporary thing to get CI past that check, could you clearly mark that commit so we don't forget to remove it again before merging?

Reply:
No, sorry. I was just trying to make the CI green but was not reflecting too much on why this was even necessary. I think at one point in the lifetime of this PR I had brought anyhow to the fuzz targets but that is no longer necessary (besides, someone else has already brought that dependency in since this PR was opened). I'll fix up the lock file instead.

Reply:
That's totally understandable!

On the other PR where you (briefly) added this exemption, I was worried we'd merged a lockfile change to `main` that somehow slipped past the cargo-vet CI checks. That's what I meant by wondering if we'd screwed up: it wouldn't be hugely surprising if either cargo-vet or the CI integration had a bug, and besides, as a community our processes around this are largely untested still. But it's nice to know that this particular case has a simpler explanation.

Reply:
Well, thanks for catching this (and the other PR!). This commit should be gone now as well as the unintential `Cargo.lock` upgrade.